packages/android_intent/android/src/main/java/io/flutter/plugins/android_intent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,85 @@
+package io.flutter.plugins.android_intent;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+import java.util.Map;
+
+/**
+ * AndroidIntentPlugin
+ */
+@SuppressWarnings("unchecked")
+public class AndroidIntentPlugin implements MethodCallHandler {
+  private final Context context;
+
+  /**
+   * Plugin registration.
+   */
+  public static void registerWith(Registrar registrar) {
+    final MethodChannel channel = new MethodChannel(registrar.messenger(), "android_intent");
```

Comment:
OK. Is this something we can hook into flutter create? I just used the default channel name given by the script.